statsmodels/tsa/holtwinters.py

```diff
@@ -304,8 +304,9 @@ class HoltWintersResults(Results):
     summary
     """
 
-    def __init__(self, model, params, **kwargs):
+    def __init__(self, model, params, fittedvalues=None, **kwargs):
```

Comment:
I'm not sure about this
in the standard pattern
fittedvalues should not be an explicit argument, because it should be a results cached attribute
in here it might be different because fittedvalues similar to statespace models is a byproduct of the estimation.